draft-ietf-quic-http.md

```diff
@@ -606,7 +605,7 @@ The following pseudo-header fields are defined for requests:
     {{Section 7.1 of SEMANTICS}}.
 
 All HTTP/3 requests MUST include exactly one value for the ":method", ":scheme",
-and ":path" pseudo-header fields, unless it is a CONNECT request; see
+and ":path" pseudo-header fields, unless the request is a CONNECT request; see
```

Comment:
RFC Editor asked:
>Because HTTP/3 requests is plural, may we update this as follows?
> 
> Original:
>>  All HTTP/3 requests MUST include exactly one value for the ":method",  
>> ":scheme", and ":path" pseudo-header fields, unless it is a CONNECT  
>> request; see Section 4.2.
> 
> Perhaps A:
>>  All HTTP/3 requests MUST include exactly one value for the ":method",  
>> ":scheme", and ":path" pseudo-header fields, except for CONNECT  
>> requests; see Section 4.2.
> 
> Perhaps B:
>>  All HTTP/3 requests MUST include exactly one value for the ":method",  
>> ":scheme", and ":path" pseudo-header fields, unless the request is a 
>> CONNECT  request; see Section 4.2.